website/docs/cli/commands/vendor/vendor-pull.mdx

```diff
@@ -108,6 +108,112 @@ The following config can be used to download the `vpc` component from an AWS pub
       version: "latest"
   ```
 
+## Vendoring from SSH
+
+This is the most common way to access non-public Git repositories from automated systems because it allows access to private repositories without interactive prompts.
+Atmos will use any installed SSH keys automatically. There are two primary ways to specify an SSH source:
+
+### SCP-style Sources
+
+Atmos supports traditional SCP-style sources, which use a colon to separate the host from the repository, like this:
+
+```shell
+git::git@github.com:cloudposse/terraform-null-label.git?ref={{.Version}}
+```
+
+Atmos rewrites this URL to the following format:
+
+```shell
+git::ssh://git@github.com/cloudposse/terraform-null-label.git?depth=1&ref={{.Version}}
+```
+
+If no username is supplied and the host is `github.com`, Atmos automatically injects the default username `git`.
+
+### Explicit SSH
+
+When the SSH scheme (`ssh://`) is explicitly specified, the URL is used as provided. In this scenario, no rewriting occurs because the URL already includes the explicit `ssh://` scheme. For example:
+
+```shell
+git::ssh://git@github.com/cloudposse/terraform-null-label.git?ref={{ .Version }}
+```
+
+the URL is used as provided. In this scenario, no rewriting occurs because the URL already includes an explicit `ssh://` schema.
+
+### Important Notes
+
+- A URL like:
```

Comment:
```suggestion
- The following URL is **invalid** because `go-getter` misinterprets `github.com:` as a URL scheme (like `http:` or `git:`), causing a parsing error:
```